grr/server/grr_response_server/gui/ui/components/timestamp/relative_timestamp_pipe.ts

```diff
@@ -0,0 +1,50 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {DatePipe} from '@angular/common';
+
+@Pipe({
+  name: 'relativeTimestamp'
+})
+export class RelativeTimestampPipe extends
+  DatePipe implements PipeTransform {
+
+  readonly oneDay = 86400000; // One day in millis
+  readonly oneHour = 3600000; // One hour in millis
+  readonly oneMin = 60000; // One minute in millis
+
+  transform(date: Date, absoluteOnly: any): any {
```

Comment:
I tried using `boolean` instead of `any`, but because the method overrides the one from `DatePipe` class, it didn't allow me. I will find a workaround.